content/docs/guides/cadvisor.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Monitoring Docker container metrics using cAdvisor
+---
+
+# Monitoring Docker container metrics using cAdvisor
+
+[cAdvisor](https://github.com/google/cadvisor) (short for **c**container **Advisor**) analyzes and exposes resource usage and performance data from running containers. cAdvisor exposes Prometheus metrics out of the box. In this guide, we will:
+
+* create a local multi-container [Docker Compose](https://docs.docker.com/compose/) installation that includes containers running Prometheus, cAdvisor, and a [Redis](https://redis.io/) server, respectively
+* examine some container metrics produced by the Redis container, collected by cAdvisor, and scraped by Prometheus
+
+## Prometheus configuration
+
+First, you'll need to [configure Prometheus](/docs/prometheus/latest/configuration/configuration) to scrape metrics from cAdvisor. Create a `prometheus.yml` file and populate it with this configuration:
+
+```yaml
+scrape_configs:
+- job_name: cadvisor
+  static_configs:
```

Comment:
Might be nice to include a `scrape_interval: 5s` in here so that people don't have to wait a long time (1 minute by default) until they can see demo data.